### DISCUSSION TITLE
Use case for CSTM-00

### DISCUSSION THREAD

**Discussion by PricedTransparently** [CMSgov/price-transparency-guide#470] [Q&A]
Hi, I am hoping to clarify the limitations of implementing CSTM-00 codes. Lets say we have this scenario:

One line of a contract specifies a number of office visit services (CPT codes) at some case rate.
The second and final line of the contract covers all CPT code procedures at some percent of billed.

In this scenario, are we able to individually report all of the billing codes from the first contract line, and then report all of the billing codes from the second line as one in-network object, with a billing_code_type of "CPT", and a billing_code of "CSTM-00"? Technically, the CSTM-00 negotiated rate would only apply to all CPT codes EXCEPT for the individual CPT codes reported in the file. Is this a correct use case for the CSTM-00 code? Or, does using a CSTM-00 code mean no other billing code of the same type should be included in the file?

In other words, can CSTM-00 codes be used as a fallback to represent any unreported billing codes, or must they be all encompassing for the given code type?

**Comment by olderthandirt5** [comment]
@shaselton-usds  We have the exact same question. We have multiple instances of this exact scenario and it would reduce the lines in our network pricing files considerably.

**Reply by olderthandirt5** [reply]
@shaselton-usds Has this question been answered elsewhere? If not, can we use CSTM-00 as a catchall for all other codes not listed?

Our situation is that we have multiple scenarios where our rates include specific prices for 4000+ CPT codes, but all others are billed at the same percentage.

Thanks, 

Bill

**Comment by Labouv0506** [comment]
Hi, I am hoping to clarify the limitations of implementing CSTM-00 codes. Lets say we have this scenario:

One line of a contract specifies a number of office visit services (CPT codes) at some case rate.
The second and final line of the contract covers all CPT code procedures at some percent of billed.

In this scenario, are we able to individually report all of the billing codes from the first contract line, and then report all of the billing codes from the second line as one in-network object, with a billing_code_type of "CPT", and a billing_code of "CSTM-00"? Technically, the CSTM-00 negotiated rate would only apply to all CPT codes EXCEPT for the individual CPT codes reported in the file. Is this a correct use case for the CSTM-00 code? Or, does using a CSTM-00 code mean no other billing code of the same type should be included in the file?

In other words, can CSTM-00 codes be used as a fallback to represent any unreported billing codes, or must they be all encompassing for the given code type?

 

This is just my two cents, would love to hear from Scott on this….

In the example sited how would you identify that it is an all but situation vs a true ALL situation consistently? It would add effort to reading and parsing the files as you would have to wrap back and reconsider how to read each time for each doctor every time this occurs or risk getting it wrong. The only way I could see this not making things more complicated and risking making mistakes in interpretation would be to create a value that indicates to look for additional exceptions…..maybe and CSMT-xx where xx is the number of codes being listed for that provider that are exceptions to the all rule, else not sure how be sure it can be understood and evaluated accurately. 

 

My understanding was this would work as stated in the second case….it means no other code of the same type should be included …else need to list out others, or if no rate applicable, not listing. 

 

Same thing applies to place of service also I believe…if you are not applying to ALL you need to list out to ensure accurate reporting. 

 

While it would potentially shrink files, wouldn’t it make being able to read and interpret MRFs consistently more challenging.  Does switching mid-stream after many done also impact this as to how to read in the future unless this is a version change etc…

 

Scott thoughts?

 

 


Leigh Anne Bouvier

Senior Business Analyst, Cost & SmartShopper Engineering

M: 727-686-7546

 E: ***@***.***


  <https://lh3.googleusercontent.com/rAauz1ClSiCzs771FkS5uDJJI29DIk2SOk9lTxNj5ovI17RgGuvVcI9ALPYQZVeRXl2J1-YUz6My-nW0JLc4SMu56sO1hcv6aUwkmHInCVIIOzPoHZtijXcSJHebK-Vw_RCcbxir> 

*As of 9/30, our emails will transition to zelis.com <http://zelis.com/>  addresses. Please contact me at ***@***.*** ***@***.***>

**Reply by ronstringfield** [reply]
I was thinking about the same, and thought maybe we could just use a CSTM-01 for all codes not covered by some other negotiated rate object for providers listed in the provider_references object?

**Comment by Labouv0506** [comment]
***@***.*** would like to recall the message,
"[CMSgov/price-transparency-guide] Use case for CSTM-00 (Discussion #470)".
